src/GraphQL.Tests/TestExtensions.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+
+namespace GraphQL.Tests
+{
+    internal static class TestExtensions
+    {
+        public static LightDictionary ToLightDictionary(this object data)
+        {
+            if (data == null)
+                return LightDictionary.Empty;
+
+            return data is ObjectProperty[] properties
+                ? new LightDictionary(properties)
```

Comment:
```suggestion
                ? properties.ToDictionary(x => x.Key, x => x.Value)
```
Why `LightDictionary`? Seems excessive.

Reply:
Done. `ToDict` because of conflict with `StringExtensions.ToDictionary`. I first made this method and class in the main project. I thought someone would need it for conversion. Since the use of `ExecutionResult.Data` directly was only in tests, I transferred it to tests.